### PR TITLE
added event handler for selecting favorite movies

### PR DESCRIPTION
**Pull request by mehmetnuricavdar**
# Description: feat/FE- event and hover added to hear icon for favorite movies

# Jira task link:https://hackyourfuture-dk.atlassian.net/browse/LANDING-23?atlOrigin=eyJpIjoiZjVhOGZjMjgxMDEzNDY3YmEyYTdkOGZmOWMyMzVhMWEiLCJwIjoiaiJ9

# Screenshots (if applicable)
Screen Recording 2023-05-12 at 12 22 31

### CODE REVIEW

File: packages/client/src/components/MovieCard/MovieCard.jsx [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+              backgroundColor: '#000000CC',
+              zIndex: 1,
+              '&:hover': {
+                color: '#FF0000',
+                cursor: 'pointer',
+              },
+            }}
+            onClick={() => toggleFavorite(props.movie)}
+          />
+        )}
```
Comment (magdazelena) [inline packages/client/src/components/MovieCard/MovieCard.jsx:89]: or you could make just the backgorundColor conditional style dependent on `isFavourite` and then you would not need to copy-paste logic :)

File: packages/client/src/components/MovieCard/useFavorites.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { useState } from 'react';
+
+const useFavorites = (initialFavorites = []) => {
```
Comment (magdazelena) [inline packages/client/src/components/MovieCard/useFavorites.js:3]: 👏

**Review by magdazelena — CHANGES_REQUESTED** [review]
one small thing :)

File: packages/client/src/components/MovieCard/MovieCard.jsx (hunk 1 above)
Comment (mehmetnuricavdar) [reply packages/client/src/components/MovieCard/MovieCard.jsx:89]: yeah, i tried to do it but unfortunately background color is taking as square, i couldn't find the way only do it inside the heart and when i checked MUI there are 2 different heart icons so I did this way but now i shortened the code and put styles in a different variable. 
Screenshot 2023-05-13 at 13 40 11

Comment (magdazelena) [reply packages/client/src/components/MovieCard/MovieCard.jsx:89]: oooh I didn't notice those were two different icons! Anyway putting the common styles into a var can be a good idea :)

**Review by magdazelena — APPROVED** [review]

**Pull request merged**